src/app/authentication/forgot-password.component.html

```diff
@@ -0,0 +1,21 @@
+<div class="col-sm-6 col-sm-offset-3">
+
+    <h1>Reset</h1>
+
+    <div class="alert alert-danger" *ngIf="errorMessage">{{errorMessage}}</div>
+    <div class="alert alert-success" *ngIf="successMessage">{{successMessage}}</div>
+
+    <form [formGroup]="resetForm" (ngSubmit)="onSubmit(resetForm.value)">
+        <div class="form-group input-group" [ngClass]="{'has-error':!resetForm.controls['email'].valid && resetForm.controls['email'].dirty}">
+            <span class="input-group-addon"><i class="glyphicon glyphicon-user"></i></span>              
+            <input type="text" class="form-control" name="email" placeholder="Email" [formControl]="resetForm.controls['email']">
```

Comment:
Oh okay! Will change the type to email. The required attribute is handled by angular.